person.rb

```diff
@@ -6,10 +6,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
 
```

Comment:
- [ ] As specified in the project requirements, kindly make sure a person `has-many` rentals as well.
This can be achieved by saying:
```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```